Rubberduck.Core/Navigation/RegexSearchReplace/RegexSearchResult.cs

```diff
@@ -7,15 +7,17 @@ namespace Rubberduck.Navigation.RegexSearchReplace
     public class RegexSearchResult
     {
         public Match Match { get; }
+
+        //FIXME: We should not save COM wrappers anywhere.
         public ICodeModule Module { get; }
         public Selection Selection { get; }
```

Comment:
Refactoring the hell out of the regex search and replace has to be its own PR. We currently need the component, because the users of the result access it all the time. I think this code is from far before switching properly to QMNs.